repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/model/CaseEventField.java

```diff
@@ -153,6 +154,19 @@ public void setPublish(Boolean publish) {
         this.publish = publish;
     }
 
+    /**
+     * whenever we want to publish this event field.
+     **/
```

Comment:
```suggestion
```
Copied but not changed from other `publish` - probably not any benefit to having a comment for these field anyway, when there is a description in the line below, so could just delete